src/current/_includes/v25.1/misc/session-vars.md

```diff
@@ -89,7 +89,7 @@
 | <a id="transaction-status"></a> `transaction_status` | The state of the current transaction. See [Transactions]({% link {{ page.version.version }}/transactions.md %}) for more details. | `NoTxn` | No | Yes |
 | <a id="transaction-timeout"></a> `transaction_timeout` | Aborts an explicit [transaction]({% link {{ page.version.version }}/transactions.md %}) when it runs longer than the configured duration. Stored in milliseconds; can be expressed in milliseconds or as an [`INTERVAL`]({% link {{ page.version.version }}/interval.md %}). | `0` | Yes | Yes |
 | <a id="troubleshooting_mode_enabled"></a> `troubleshooting_mode_enabled` | When enabled, avoid performing additional work on queries, such as collecting and emitting telemetry data. This session variable is particularly useful when the cluster is experiencing issues, unavailability, or failure. | `off` | Yes | Yes |
-| <a id="use_declarative_schema_changer"></a> `use_declarative_schema_changer` | Whether to use the declarative schema changer for supported statements. See [Declarative schema changer]({% link {{ page.version.version }}/online-schema-changes.md %}#declarative-schema-changer) for more details. | `on` | Yes | Yes |
+| <a id="use_declarative_schema_changer"></a> `use_declarative_schema_changer` | Whether to use the declarative schema changer for supported statements. | `on` | Yes | Yes |
```

Comment:
This still says declarative schema changer, for obvious reasons. Just double-checking that leaving it in is intentional.

Reply:
Yes, intentional, thanks for checking! we're not renaming any settings nor removing any stuff from release notes.  The machinery called "declarative schema changer" still exists internally and may matter "sometimes", but not usually for most users. If customers have schema change issues it may be necessary for support etc to engage with them re: that term, but we're just deprioritizing talking about it so much in normie user-facing docs re: schema changes